src/js/angular-tooltips.js

```diff
@@ -26,7 +26,7 @@
           , width
           , offsetTop
           , offsetLeft
-          , title = attr.title || ''
+          , title = attr.tooltipTitle || ''
```

Comment:
I change it, because title is the attribute of html. If I use title, the default title will be showing at the same time of tooltip showing.  For example,
![image](https://cloud.githubusercontent.com/assets/2586026/5787423/b7a5458c-9e38-11e4-9123-789f2cec63f5.png)

It's really annoying.